routing.rst

```diff
@@ -953,7 +953,7 @@ and in route imports. Symfony defines some special attributes with the same name
                 format="html">
 
                 <requirement key="_locale">en|fr</requirement>
-                <requirement key="_format">html|rss</requirement>
+                <requirement key="_format">html|xml</requirement>
```

Comment:
In the "Annotations" and "YAML" examples, the _format parameter is set to html | xml.